autorandr.py

```diff
@@ -1178,7 +1187,7 @@ def main(argv):
             exec_scripts(profile_folder, "postsave", {
                 "CURRENT_PROFILE": options["--save"],
                 "PROFILE_FOLDER": profile_folder,
-                "MONITORS": ":".join(config.keys()),
+                "MONITORS": ":".join(enabed_monitors(load_config)),
```

Comment:
This doesn't look right. Shouldn't this be `enabled_monitors(config)`?

Reply:
Yes you're right; it's fixed now.